server/service/integration_enterprise_test.go

```diff
@@ -6278,112 +6320,6 @@ func (s *integrationEnterpriseTestSuite) TestAllSoftwareTitles() {
 	}, []fleet.SoftwareTitle{*stResp.SoftwareTitle})
 }
 
-func (s *integrationEnterpriseTestSuite) TestListHostUpcomingActivities() {
```

Comment:
I think this was just moved to core integration tests, right? Any reason we keep some here and others there?

Reply:
Yeah, I moved this one to core tests, but I think I didn't move the others as there are many more and I didn't want to make the diff in this PR huge. I can move them though, this diff is already fairly large @mna what do you think?

Reply:
As you wish, I don't mind leaving them in the enterprise tests as that will allow us to easily add team-related tests to those scenarios (which requires a premium license).